shapes/Rectangle.cpp

```diff
@@ -4,6 +4,15 @@
 Rectangle::Rectangle(double x, double y, Color c) : x_(x), y_(y),  Shape(c)
 {}
 
+Rectangle::Rectangle() : x_(1), y_(1)
+{}
+
+
+Rectangle::Rectangle(Color c) : Rectangle()
```

Comment:
we want to have such 2 constructors:
`Rectangle(double x, double y, Color c);  // already implemented`
`Rectangle(double x, double y);  // not implemented`
The latter should call the first one on it's initializer list.